gamestonk_terminal/papermill/due_diligence_view.py

```diff
@@ -1,11 +1,22 @@
+"""Papermill Due Diligence View Module"""
+__docformat__ = "numpy"
+
 import argparse
+from typing import List
 from datetime import datetime
 import os
 import webbrowser
 import papermill as pm
 
 
-def analysis(l_args):
+def due_diligence(other_args: List[str]):
```

Comment:
Why the name "other_args"? Since these are the main args used in this function

Reply:
I think we go
input -> <menu item, e.g. dd> <more args - other_args>

It was l_args originally, but in a since we don't need l_ with type hints, and args sounds too generic plus we already chopped dd off the original input, I thought I'd call it other_args.

Reply:
Got you now. Yea, removing the "l_" only makes sense. Was just wondering about "other", sounds good to me. Will start following this convention